frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -141,7 +141,7 @@ export function PrintTestDeckScreen(): JSX.Element {
   const currentUserType = currentUserSession.type;
   const { election, electionHash } = electionDefinition;
   const [precinctIds, setPrecinctIds] = useState<string[]>([]);
-  const [printIndex, setPrintIndex] = useState<PrintIndex | undefined>();
+  const [printIndex, setPrintIndex] = useState<PrintIndex | null>();
```

Comment:
Once again, I have a slight preference for `null` over `undefined` as an explicitly assigned empty value

Reply:
One advantage of using `undefined` is that you can just do this:

```suggestion
  const [printIndex, setPrintIndex] = useState<PrintIndex>();
```